components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/information/InformationMessageCardView.kt

```diff
@@ -0,0 +1,178 @@
+
+    HmrcCardView(modifier = modifier, customBackgroundColor = backgroundColor) {
+        Row(
+            modifier = modifier
+                .fillMaxWidth()
+                .enableTalkBackMergeAccessibility()
+                .padding(HmrcTheme.dimensions.hmrcSpacing16),
+        ) {
+            Image(
+                modifier = Modifier.size(HmrcTheme.dimensions.hmrcIconSize36),
+                painter = painterResource(id = icon),
+                contentDescription = "",
+                colorFilter = ColorFilter.tint(iconColor)
+            )
+            Spacer(modifier = Modifier.width(HmrcTheme.dimensions.hmrcSpacing8))
+            Text(
+                text = headline,
+                modifier = Modifier
+                    .weight(1f)
+                    .heightIn(HmrcTheme.dimensions.hmrcIconSize36)
+                    .wrapContentSize(align = Alignment.CenterStart)
+                    .semantics {
+                        this.contentDescription = headlineContentDescription
+                    },
+                style = HmrcTheme.typography.h6,
+                color = textColor
+            )
+        }
+        if (text != null) {
+            Row(
+                modifier = modifier
+                    .fillMaxWidth()
+                    .padding(
+                        start = HmrcTheme.dimensions.hmrcSpacing16,
+                        end = HmrcTheme.dimensions.hmrcSpacing16,
+                        bottom = HmrcTheme.dimensions.hmrcSpacing16
+                    )
+            ) {
+                Text(
+                    text = text,
+                    modifier = Modifier,
+                    style = HmrcTheme.typography.body,
+                    color = textColor
+                )
+            }
+        }
+        if (buttons.isNotEmpty()) {
+            buttons.forEach {
+                it.invoke()
+                Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing8))
+            }
+            Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing8))
+        }
+    }
+}
+
+class InformationMessageButton(
+    val text: String,
+    val messageType: MessageType,
+    val buttonType: ButtonType = ButtonType.ACTION,
+    val onClick: () -> Unit
+) {
+    @Composable
+    operator fun invoke() {
+        val (colors, border) = when (buttonType) {
+            ButtonType.ACTION -> {
+                Pair(
+                    ButtonDefaults.buttonColors(
+                        containerColor = HmrcTheme.colors.hmrcWhite, contentColor = HmrcTheme.colors.hmrcBlue
+                    ),
+                    null
+                )
+            }
+            ButtonType.OUTLINE -> {
+                Pair(
+                    ButtonDefaults.buttonColors(
+                        containerColor = Color.Transparent, contentColor = messageType.headlineTint
+                    ),
+                    BorderStroke(width = 1.0.dp, color = messageType.headlineTint,)
```

Comment:
Do we need to define the button stroke on the `Dimensions` file?

Reply:
That doesn't seem to change anything?

Reply:
Yes, that's correct. There's nothing to alter; it's likely that the rendering results between XML and Compose are slightly different.